tests/test_pod_exec.py

```diff
@@ -8,21 +8,27 @@
 def main():
     config.load_kube_config()
     v1 = client.CoreV1Api()
-    ret = v1.list_pod_for_all_namespaces(watch=False,
-                                        label_selector='app=rook-ceph-tools')
+    ret = v1.list_pod_for_all_namespaces(
+        watch=False, label_selector='app=rook-ceph-tools'
+    )
+
     for i in ret.items:
         namespace = i.metadata.namespace
         name = i.metadata.name
         break
-    cmd = "ceph osd df"
-    po = pod.Pod(name, namespace)
-    out, err, ret = po.exec_command(cmd=cmd, timeout=20)
+
+    cmd = f"ceph osd df"
+    po = pod.Pod(
```

Comment:
We only need to split these sorts of lines up if they exceed our 80 char limit per line. If they are short enough like this one here to be on one line, then they should remain as such.